kickoff/views/submit.py

```diff
@@ -8,15 +8,38 @@
 from kickoff.model import getReleaseTable
 from kickoff.views.forms import FennecReleaseForm, FirefoxReleaseForm, \
     ThunderbirdReleaseForm, getReleaseForm
+import urllib2
+import json
 
 log = logging.getLogger(__name__)
 
 
+def retrieveADI():
+    defaultURL = "http://localhost/"
+    # Download the source of the data
+    # Download it from a remote interface to avoid a dependency on vertica db
+    # We do that in Python to avoid the Cross Origin complexity in JS
+    try:
+        response = urllib2.urlopen(defaultURL + 'partial-beta.json')
+        jsonSourceBeta = response.read().strip()
+        response = urllib2.urlopen(defaultURL + 'partial-release.json')
+        jsonSourceRelease = response.read().strip()
+        response = urllib2.urlopen(defaultURL + 'partial-esr.json')
+        jsonSourceESR = response.read().strip()
+        return jsonSourceBeta, jsonSourceRelease, jsonSourceESR
+    except urllib2.HTTPError, e:
```

Comment:
Yes but as mentioned in the comment, if the json files are remote, this requires the Cross Origin thing to be set up... This was the easier way (for me) to do it.